src/services/auth.ts

```diff
@@ -19,23 +25,30 @@ export const auth = {
             }
         );
     },
-    async getAccountID() {
-        const details = window.localStorage.getItem(authDetailsKey);
+    getAuthDetails() {
+        const session = window.localStorage.getItem(sessionStorageKey);
+        let response: AuthDetails | null = null;
 
-        if (details) {
-            return JSON.parse(details).account_id;
-        } else {
-            return null;
-        }
-    },
-    async getToken() {
-        const details = window.localStorage.getItem(authDetailsKey);
+        if (session) {
+            const user = window.localStorage.getItem(userStorageKey);
 
-        if (details) {
-            return JSON.parse(details).token;
-        } else {
-            return null;
+            response = { session: JSON.parse(session) };
+
+            if (user) {
+                response.user = JSON.parse(user);
+            }
         }
+
+        return response;
+    },
```

Comment:
We store session and use in different slots in storage but we return them together. That way there is just a single call to fetch.